build.gradle

```diff
@@ -80,10 +80,8 @@ dependencies {
 	implementation 'org.fusesource.jansi:jansi:2.3.4'
 	implementation 'org.json:json:20210307'
 	implementation 'org.mozilla:rhino:1.7.13'
-	implementation 'org.swinglabs:swingx:1.0'
+	implementation 'org.swinglabs:swingx:1.6'
 	implementation 'org.tmatesoft.svnkit:svnkit:1.10.3'
-
-	toCopy 'org.swinglabs:swingx:1.0'
```

Comment:
Do we no longer need to copy it?

Reply:
This target was always intended for downloading jars into the src/jar directory.

Since Gradle is bundling our jars for us (as opposed to Ant grabbing all the jars in a directory), we no longer need this target.